### PR TITLE
edit upgrade-monitoring-services.md

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

This PR synchronizes the new changes made in https://github.com/pingcap/docs/pull/14523.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: new changes in https://github.com/pingcap/docs/pull/14523
- Other reference link(s):  https://github.com/pingcap/docs-cn/pull/14510

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qqqdan — APPROVED** [review]

File: upgrade-monitoring-services.md [hunk 1]
```diff
@@ -7,15 +7,16 @@ summary: 介绍如何升级 TiDB 集群监控组件 Prometheus、Grafana 和 Ale
 
 使用 TiUP 部署 TiDB 集群时，TiUP 会同时自动部署 Prometheus、Grafana 和 Alertmanager 等监控组件，并且在集群扩容中自动为新增节点添加监控配置。通过 TiUP 自动部署的监控组件并不是这些三方组件的最新版本，如果你需要使用最新的三方组件，可以按照本文的方法升级所需的监控组件。
 
-需要注意的是，TiUP 会使用自己的配置参数覆盖监控组件的配置。如果你直接通过替换监控组件配置文件的方式升级监控组件，在之后对集群进行 `deploy`、`scale-out`、`scale-in`、`reload` 等操作时，该升级可能被 TiUP 所覆盖，导致升级出错。如果需要升级 Prometheus、Grafana 和 Alertmanager，请使用本文介绍的升级步骤。
+当管理集群时，TiUP 会使用自己的配置参数覆盖监控组件的配置。如果你直接通过替换监控组件配置文件的方式升级监控组件，在之后对集群进行 `deploy`、`scale-out`、`scale-in`、`reload` 等 TiUP 操作时，该升级可能被 TiUP 所覆盖，导致升级出错。如果需要升级 Prometheus、Grafana 和 Alertmanager，请参考本文介绍的升级步骤，而不是直接替换配置文件。
 
 > **注意：**
 >
-> - 如果监控组件不是由 TiUP 部署和管理，可以直接升级监控组件，无需参考本文档。
+> - 如果现有的监控组件是[手动部署](deploy-monitoring-services.md)的，而不是由 TiUP 部署的，你可以直接升级监控组件，无需参考本文。
```
Comment (qiancai) [inline upgrade-monitoring-services.md:14]:
```suggestion
> - 如果现有的监控组件是[手动部署](/deploy-monitoring-services.md)的，而不是由 TiUP 部署的，你可以直接升级监控组件，无需参考本文。
```

File: upgrade-monitoring-services.md [hunk 2]
```diff
@@ -62,7 +63,8 @@ tiup cluster patch <cluster-name> prometheus-{new-version}.tar.gz -R prometheus
 
 ### 第 1 步：从 Grafana 官网的下载新版本安装包
 
-从 [Grafana 官网下载页面](https://grafana.com/grafana/download?pg=get&plcmt=selfmanaged-box1-cta1)下载组件安装包，并解压。
+1. 从 [Grafana 官网下载页面](https://grafana.com/grafana/download?pg=get&plcmt=selfmanaged-box1-cta1)下载组件安装包。你可以根据需要选择下载 `OSS`版或 `Enterprise` 版。
```
Comment (hfxsd) [inline upgrade-monitoring-services.md:66]:
```suggestion
1. 从 [Grafana 官网下载页面](https://grafana.com/grafana/download?pg=get&plcmt=selfmanaged-box1-cta1)下载组件安装包。你可以根据需要选择下载 `OSS` 版或 `Enterprise` 版。
```

File: upgrade-monitoring-services.md [hunk 3]
```diff
@@ -108,3 +110,5 @@ TiDB 安装包中直接使用了 AlertManager 官方组件包，因此升级 Ale
 ```bash
 tiup cluster patch <cluster-name> alertmanager-v{new-version}-linux-amd64.tar.gz -R alertmanager
 ```
+
+升级完成后，可以打开 AlertManager 主页（地址通常是 `http://<Alertmanager-server-host-name>:9093`），点击顶部导航菜单中的 **Status**，然后查看 Alertmanager 的版本信息，确认升级成功。
```
Comment (hfxsd) [inline upgrade-monitoring-services.md:114]: L98、L100 的三处 AlertManager 也都需要改成 Alertmanager 
```suggestion
升级完成后，可以打开 Alertmanager 主页（地址通常是 `http://<Alertmanager-server-host-name>:9093`），点击顶部导航菜单中的 **Status**，然后查看 Alertmanager 的版本信息，确认升级成功。
```

**Review by hfxsd — APPROVED** [review]

File: upgrade-monitoring-services.md (hunk 3 above)
Comment (qiancai) [reply upgrade-monitoring-services.md]: updated in https://github.com/pingcap/docs-cn/pull/14836/commits/7ef6eadd176cba3435a6d7a3013a0257957dd21f

**Pull request merged**